notebooks/official/ml_metadata/vertex-pipelines-ml-metadata.ipynb

```diff
@@ -779,9 +750,22 @@
         "id": "u-iTnzt3B6Z_"
       },
       "source": [
-        "### Start two pipeline runs\n",
+        "### Initiate pipeline runs\n",
         "\n",
-        "Next you'll kick ofg **two** runs of our pipeline. First, define a timestamp to use for our pipeline job IDs:"
+        "First, define a timestamp to use as your pipeline job IDs.\n",
+        "\n",
+        "Then for each run, create an instance of `PipelineJob` from the `pipeline_jobs` module.\n",
+        " For each instance provide the following details:\n",
+        "\n",
+        "* `display_name` : Human-readable name for the pipeline job.\n",
+        "* `template_path` : This specifies the path to the pipeline template file in JSON format, which contains the pipeline's configuration and structure created in the previous steps.\n",
+        "* `job_id` : This sets a unique identifier for the job.\n",
+        "* `parameter_values` : This dictionary contains key-value pairs for the parameters required by the pipeline which are metioned during pipeline definition.\n",
+        "    * `bq_table` :  Specifies the BigQuery table to use.\n",
+        "    * `output_data_path` : Defines the path for the output data file.\n",
+        "    * `project` : Specifies the Google Cloud project ID.\n",
+        "    * `region` : Defines the region where the pipeline will run.\n",
+        "* enable_caching : When set to `True`, caching is enabled for the pipeline run. This allows the system to reuse previous results if the same job is executed before with identical parameters, thereby saving time and resources"
```

Comment:
- change: " This allows the system to reuse previous results if the same job is executed before with identical parameters, thereby saving time and resources"
    - "This lets the system reuse previous results, if the same job has been executed before with identical parameters, thereby saving time and resources""